src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -97,7 +97,7 @@
  * </p>
  * <pre>
  * &lt;module name="SummaryJavadocCheck"&gt;
- *   &lt;property name="period" value="&#12290;"/&gt;
```

Comment:
This change to fix https://bugs.openjdk.java.net/browse/JDK-8148480

Reply:
it is addressed in https://github.com/checkstyle/checkstyle/pull/7266